src/stepFunctions/asl/asl-yaml-languageservice.ts

```diff
@@ -0,0 +1,249 @@
+/*!
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* eslint-disable no-useless-escape */
+/* eslint-disable @typescript-eslint/unbound-method*/
+
+import {
+    ASL_SCHEMA,
+    doCompleteAsl,
+    ClientCapabilities,
+    getLanguageService as getASLLanguageService,
+    LanguageService,
+} from 'amazon-states-language-service'
+import {
+    DocumentLanguageSettings,
+    JSONDocument,
+    getLanguageService as getLanguageServiceVscode,
+    LanguageServiceParams,
+} from 'vscode-json-languageservice'
+import {
+    parse as parseYAML,
+    YAMLDocument,
+    SingleYAMLDocument,
+} from 'yaml-language-server/out/server/src/languageservice/parser/yamlParser07'
+import {
+    CompletionItem,
+    CompletionList,
+    DocumentSymbol,
+    FormattingOptions,
+    Hover,
+    TextEdit,
+    TextDocument,
+    Position,
+    Range,
+    SymbolInformation,
+} from 'vscode-languageserver-types'
+import { matchOffsetToDocument } from 'yaml-language-server/out/server/src/languageservice/utils/arrUtils'
+import { YAMLSchemaService } from 'yaml-language-server/out/server/src/languageservice/services/yamlSchemaService'
+import { YAMLCompletion } from 'yaml-language-server/out/server/src/languageservice/services/yamlCompletion'
+import * as URL from 'url'
+import * as prettier from 'prettier'
```

Comment:
Yep, formatting on a .asl.yaml doc still works

Reply:
To follow up on what @awschristou said, have you tried building the VSIX (use `npm run package`), installing that, and testing on the installed VSIX? This might work through a local Extension Development Host since it's already on your machine as a devDependency; installing the VSIX and testing that should only build in actual dependencies.